gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -79,6 +79,19 @@
      * "dimensions = (string) dim1:dim2:dim3:dim4, dim1:dim2:dim3:dim4"
      */
 
+/**
+ * @brief Default static capibility for other/tensor
+ */
+#define GST_FLATBUF_TENSOR_CAP_DEFAULT \
+    "other/flatbuf-tensor, " \
+    "framerate = " GST_TENSOR_RATE_RANGE
+    /**
+     * type should be one of types in GST_TENSOR_TYPE_ALL
+     * "type = (string) uint8"
+     * dimension shoule be a formatted string with rank NNS_TENSOR_RANK_LIMIT
+     * "dimension = (string) dim1:dim2:dim3:dim4"
+     */
+
```

Comment:
I'm not sure if this is the right location. I want to hear reviewer's opinion.

Reply:
if you assume that each gstbuf instance will always have a single frame, that'd be fine. If you need to assume that there might be multiple frames in it, you need to consult with those who have written audio converters.